README.md

```diff
@@ -2,31 +2,32 @@
 
 Thank you for choosing us to guide you through your journey to become a software developer! This **Foundations Course** is for students who have enrolled in either a day or evening Web Development cohort.
 
-We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this couse to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
+We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this course to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
 
 ## Foundations Course is Mandatory
 
-If you don't do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
+If you do not do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
 
-Nashville Software School is an accelerated bootcamp, and you will hit the ground running on day one. We will assume that you've completed this work, and provide minimal resources to you if you have not. Additionally, the life of a software developer is one of focus and meeting committments. Completing the **Foundations Course** shows us that you have the ability to prioritize this work, can focus on it, and meet the committment of completing it.
+Nashville Software School is an accelerated Bootcamp, and you will hit the ground running on day one. We will assume that you have completed this work and provide minimal resources if you have not. Additionally, the life of a software developer is one of focus and meeting commitments. Completing the **Foundations Course** shows us that you can prioritize this work, focus on it, and meet the commitment of completing it.it.
```

Comment:
`Bootcamp` should be changed to `boot camp`. Also, you have an extra `it.` at the end of line 11.